concerned-coyotes/earlyinternet/utils/helpers.py

```diff
@@ -0,0 +1,10 @@
+def create_url_with_parameters(base_url: str, params: dict) -> str:
```

Comment:
The name of the function is misleading as it adds a query not parameters to an url.